src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java

```diff
@@ -30,35 +31,77 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
+                        "again next year and follow us on our social media channels for future programs. " +
+                        "If you have any clarifications, please reach out to us via " +
+                        "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a>";
+
+                emailService.sendEmail(mentor.getProfile().getEmail(), StringUtils.capitalize(mentor.getState().name()), message, false);
+
+            }
         }
     }
 
     public void sendMenteeSelectionEmails(long id, Optional<Program> program) throws IOException, MessagingException {
         List<Mentor> approvedMentors = mentorRepository.findAllByProgramIdAndState(id, EnrolmentState.APPROVED);
 
-        String message = "You can approve or reject your mentees by visiting the dashboard";
         for (Mentor mentor : approvedMentors) {
-            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message);
+
+            String message = "Dear " + mentor.getProfile().getFirstName() + ",<br /><br />" +
+                    "You have student applications waiting to be reviewed. You can approve or reject your mentees " +
+                    "by visiting the <b>ScholarX dashboard.</b>";
+
+            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message, true);
         }
     }
 
     public void sendOnGoingEmails(long id, Optional<Program> program) throws IOException, MessagingException {
         List<Mentor> approvedMentors = mentorRepository.findAllByProgramIdAndState(id, EnrolmentState.APPROVED);
 
-        String message = "You can check your mentees by visiting the dashboard";
         for (Mentor mentor : approvedMentors) {
-            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message);
+
+            String message = "Dear " + mentor.getProfile().getFirstName() + ",<br /><br />" +
+                    "<b>Congratulations!</b><br />Students have accepted you as their mentor. " +
+                    "You can check your mentees and their contact details by visiting the <b>ScholarX dashboard.</b> " +
+                    "Please make the first contact with them as we have instructed them to wait for your email.";
+
+            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message, true);
         }
     }
 
     public void sendMentorConfirmationEmails(long id, Optional<Program> program) throws IOException, MessagingException {
         List<Mentee> mentees = menteeRepository.findAllByProgramId(id);
 
-        String message = "You can check your mentor by visiting the dashboard";
+
         for (Mentee mentee : mentees) {
-            emailService.sendEmail(mentee.getProfile().getEmail(), program.get().getTitle(), message);
+            String message = "Dear " + mentee.getProfile().getFirstName() + ",<br /><br />" +
```

Comment:
Is this the correct message? When the program is moved to the `MENTOR_CONFIRMATION` state, the mentees with multiple mentors can confirm one mentor. Mentees with only one mentor don't have to do anything.
(An email with this content can be sent to all mentees when the program state changes to `MENTEE_SELECTION`)

cc: @anjula-sack @Gravewalker666